Makefile

```
@@ -0,0 +1,7 @@
+PHONY: build
+build:
```

`.PHONY` ですかね？
https://yu-nix.com/archives/makefile-phony/
```suggestion
.PHONY: build
build:
```

---

間違えました！修正します